<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCNWindow</class>
 <widget class="QMainWindow" name="CCNWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1470</width>
    <height>935</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CCNControl</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(110, 0, 165);
border:1px solid rgb(0, 170, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>1301</width>
      <height>741</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border:0px solid blue;</string>
    </property>
    <property name="text">
     <string>Visual CCN DBMS</string>
    </property>
   </widget>
   <widget class="CCNButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>1</y>
      <width>39</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
selection-background-color: rgb(255, 0, 0);
border:0px;</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>1</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
selection-background-color: rgb(255, 0, 0);
border:0px;</string>
    </property>
    <property name="text">
     <string>□</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>1</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
selection-background-color: rgb(255, 0, 0);
border:0px;</string>
    </property>
    <property name="text">
     <string>－</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CCNButton</class>
   <extends>QPushButton</extends>
   <header location="global">ccnbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
